postcode_lookup/templates/includes/single_date.html

```diff
@@ -4,9 +4,9 @@ <h2 id="date-{{ date.date_data.date }}">
     {% endif %}
 </h2>
 
-{% if date.date_data.ballots|length > 1 %}
+{% if date.uncancelled_ballot_count > 1 %}
     <p>
-        {% trans count=date.date_data.ballots|length %}
+        {% trans count=date.date_data.uncancelled_ballot_count %}
             You will have {{ count }} ballot papers to fill out:
         {% endtrans %}
     </p>
```

Comment:
Lets say you have 1 cancelled ballot and 2 un-cancelled.
We're going to say "You will have 2 ballot papers to fill out"
and then we're going to show you a `<ul>` with a list of 3 things.
That's kinda wierd.

Then if you have 1 ballot happening and some number of cancelled ballots, we'll omit this section completely but show you details about multiple ballots anyway.

I think if we're going to show you details of more than one thing, it probably makes sense to show this section.

We could only show the un-cancelled ballots in the `<ul>` (so the number of things in the list matches the number)
We could also consider showing the full list but with an indicator of any that are not going ahead (accepting that finding a word for this is hard).

Reply:
Yeah, I think this is less bad than at the moment, but I'm happy to change the way it works more. 

I don't like breaking the link between the table of contents and the headings, but I can add `(postponed)` after the TOC item and try to add some text about it in the intro text.

Reply:
Yeah, but we have logic that sort of deals with this already:

https://github.com/DemocracyClub/ec-postcode-lookup-pages/blob/10a4d05244edd19838159974659f5809f481dcd3/postcode_lookup/template_sorter.py#L390

I think it's ok to use that as a starting point

Reply:
Ah - yes nice :+1: